node/main.ts

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env -S deno run --unstable -A
```

Comment:
Can we rename this to cli.ts?

Reply:
For context, it matters for `deno install`.

If this is named cli.ts then it will take the parent directory as the name, meaning deno install https://deno.land/std/node installs this shim as `node`.

Reply:
That works for `main.ts` as well. I also prefer `cli.ts` though, `main.ts` is normally an entrypoint for the surrounding application code whereas this is more quirky than that; it runs separate user code.